packages/js-runtime/src/index.ts

```diff
@@ -28,7 +29,26 @@ import './runtime/http/fetch';
 //
 // NOTE: we use `var` to that we can refer to these variables
 // using `globalThis.VARIABLE`.
+
+interface CompressionStream {
+  readonly readable: ReadableStream<Uint8Array>;
+  readonly writable: WritableStream<Uint8Array>;
+}
+
+interface DecompressionStream {
+  readonly readable: ReadableStream<Uint8Array>;
+  readonly writable: WritableStream<Uint8Array>;
+}
+
+type CompressionFormat = 'deflate' | 'deflate-raw' | 'gzip';
+
 declare global {
+  // eslint-disable-next-line @typescript-eslint/no-namespace
+  namespace globalThis {
+    var CompressionStream: { new (format: CompressionFormat): CompressionStream; prototype: CompressionStream };
+    var DecompressionStream: { new (format: CompressionFormat): DecompressionStream; prototype: DecompressionStream };
+  }
```

Comment:
These types already exist inside `lib.webworker.d.ts` so we should be able to remove them

Reply:
done